tests/yaml_comment/end-comment.yml

```diff
@@ -0,0 +1,13 @@
+parent:
+  one: 1
+  # two: 2
+
+a:
+  b:
+   #b
+ #a
+
+A:
+  B:
+ #A
+   #A
```

Comment:
Didn’t these have to be at the end of the file to reproduce the bug?

Reply:
It just needs to be at the end of a collection.